wiki/Manual.md

```diff
@@ -4243,7 +4243,9 @@ exchange.create_limit_buy_order (symbol, amount, price[, params])
 exchange.create_limit_sell_order (symbol, amount, price[, params])
 ```
 
-#### Stop Orders
+#### Conditional Orders
+
+<a name="Stop Orders" id="Stop Orders"></a><a name="Trigger Orders" id="Trigger Orders"></a>
```

Comment:
I think we are over and over linking & directing users to "STOP ORDERS" while we 'are trying' to avoid & obsolete that term. So, it's better we no longer endorse that term in bolds